pkg/controller/vitesskeyspace/vitesskeyspace_controller.go

```diff
@@ -179,6 +184,23 @@ func (r *ReconcileVitessKeyspace) Reconcile(request reconcile.Request) (reconcil
 	topoResult, err := r.reconcileTopology(ctx, vtk)
 	resultBuilder.Merge(topoResult, err)
 
+	wr, err := newWrangler(ctx, *vtk)
+	if err != nil {
+		r.recorder.Eventf(vtk, corev1.EventTypeWarning, "StatusUpdateWarning", "failed to retrieve resharding information: %v", err)
+	}
+	if wr != nil {
+		workflows, err := wr.ListAllWorkflows(ctx, vtk.Name)
+		if err != nil {
+			r.recorder.Eventf(vtk, corev1.EventTypeWarning, "StatusUpdateWarning", "failed to list active resharding workflows: %v", err)
+		}
+		if len(workflows) != 0 {
+			vtk.Status.ReshardingInProgress = true
+			vtk.Status.ActiveWorkflows = workflows
+		} else {
+			vtk.Status.ReshardingInProgress = false
+		}
```

Comment:
Cleanliness suggestion:
```suggestion
		vtk.Status.ReshardingInProgress = len(workflows) != 0
		vtk.Status.ActiveWorkflows = workflows
```

Reply:
Nice suggestion, done!